### PR TITLE
docs(v2): SEO

### PR DESCRIPTION
**Pull request by wgao19**
## Motivation

Addressing #1640, refactor & improve docs for SEO

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/Docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Bonus points for screenshots and videos!)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)

### CODE REVIEW

File: website/docs/reaching-users.md [hunk 1]
```diff
@@ -17,20 +21,93 @@ References
...
+
+### Customize SEO
+
+To add customized SEO, use the `Head` component from `@docusaurus/HEAD`.
+
+```js
+import Head from '@docusaurus/HEAD';
+
+const MySEO = () => (
+  <React.Fragment>
```
Comment (yangshun) [inline website/docs/reaching-users.md]: Why not use `<>`?

File: website/docs/reaching-users.md [hunk 2]
```diff
@@ -17,20 +21,93 @@ References
...
+### Page specific SEO
+
+To add SEO to your doc, use the following fields in your doc's front matter:
+
+```yaml
+---
+keywords:
+  - docs
+  - docusaurus
+description: 'How do I find you when I cannot solve this problem'
```
Comment (yangshun) [inline website/docs/reaching-users.md]: I don't think yaml needs quotes.

File: website/docs/reaching-users.md [hunk 3]
```diff
@@ -17,20 +21,93 @@ References
...
+   * - `${title} · ${tagline}`
+   * - `Image for ${title} · ${tagline}`
+   */
+  title: 'Docusaurus',
+  tagline: 'Easy to Maintain Open Source Documentation Websites',
+  /**
+   * The following fields are used for `og:image` and `twitter:image`
+   */
+  baseUrl: '/',
+  siteUrl: 'https://docusaurus.io/',
```
Comment (yangshun) [inline website/docs/reaching-users.md]: Don't need trailing slash

File: website/docs/reaching-users.md [hunk 4]
```diff
@@ -17,20 +21,93 @@ References
...
+`image` is used by search engines and Twitter for a cover or thumbnail image when displaying the link to your post.
+
+Note that the file of this image cannot be SVG.
+
+### Customize SEO
+
+To add customized SEO, use the `Head` component from `@docusaurus/HEAD`.
+
+```js
+import Head from '@docusaurus/HEAD';
```
Comment (yangshun) [inline website/docs/reaching-users.md]: `@docusaurus/Head`

File: website/docs/reaching-users.md [hunk 5]
```diff
@@ -17,20 +21,93 @@ References
...
+
+- Type: `string`
+
+`image` is used by search engines and Twitter for a cover or thumbnail image when displaying the link to your post.
+
+Note that the file of this image cannot be SVG.
+
+### Customize SEO
+
+To add customized SEO, use the `Head` component from `@docusaurus/HEAD`.
```
Comment (yangshun) [inline website/docs/reaching-users.md]: `@docusaurus/Head`

File: website/docs/reaching-users.md (hunk 1 above)
Comment (wgao19) [reply website/docs/reaching-users.md]: Updated 🦖

**Pull request merged**